generators/run/templates/run

```diff
@@ -338,7 +338,7 @@ case $run_command in
         if [ -n "${containers_using_volume}" ]; then docker rm --force ${containers_using_volume}; fi
         docker volume rm --force ${node_cache_volume}
     ;;
-<% if (django) { %>    "django") django_run "${project}-django-$(date +'%s')" "${@}" "--rm" ;;
-<% } %>    "node") node_run "${project}-node-$(date +'%s')" "${@}" ;;
+<% if (django) { %>    "django") django_run "${project}-django-$(date +'%s')" "'${*}'" "--rm" ;;
+<% } %>    "node") node_run "${project}-node-$(date +'%s')" "'${*}'" ;;
```

Comment:
I get:

``` bash
$ ./run node yarn run serve examples
docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"'yarn\": executable file not found in $PATH".
ERRO[0000] error getting events from daemon: context canceled 
```

Unless I remove the extra `'`s. Then it works. (`"'${*}'"` -> `"${*}"`)